openmrs/src/main/java/org/motechproject/openmrs/tasks/impl/OpenMRSActionProxyServiceImpl.java

```diff
@@ -95,7 +95,7 @@ public Encounter createEncounter(String configName, DateTime encounterDatetime,
         //While creating observations, the encounterDateTime is used as a obsDateTime.
         List<Observation> observationList = MapUtils.isNotEmpty(observations) ? convertObservationMapToList(observations, encounterDatetime) : null;
 
-        EncounterType type = new EncounterType(encounterType);
+        EncounterType type = new EncounterType(encounterType, null);
```

Comment:
it should be new EncounterType(null, encounterType). Uuid is a second parameter in constructor